src/Passage.sol

```diff
@@ -56,6 +68,40 @@ contract Passage {
         emit Enter(rollupChainId, address(0), rollupRecipient, msg.value);
     }
 
+    /// @notice Allows a special transaction to be sent to the rollup with sender == L1 msg.sender.
+    /// @dev Transaction is processed after normal rollup block execution.
+    /// @param rollupChainId - The rollup chain to send the transaction to.
+    /// @param to - The address to call on the rollup.
+    /// @param data - The data to send to the rollup.
+    /// @param value - The amount of Ether to send on the rollup.
+    /// @param gas - The gas limit for the transaction.
+    /// @param maxPrioFee - The maximum priority fee for the transaction.
+    /// @param maxBaseFee - The maximum base fee for the transaction.
+    /// @custom:emits Transact indicating the transaction to mine on the rollup.
+    function transact(
+        uint256 rollupChainId,
+        address to,
+        bytes calldata data,
+        uint256 value,
+        uint256 gas,
+        uint256 maxPrioFee,
+        uint256 maxBaseFee
```

Comment:
per linear discussion, my list here was slightly wrong. instead of `maxBase` it should be `maxFeePerGas`.

Given that these transactions aren't sequenced by a sequencer, we can/should remove `maxPrio`.  that lets us simplify the interface a bit

Reply:
https://github.com/init4tech/zenith/pull/45/commits/8145d2cf61edcd3839de8bdb0d92ec9d172ff36b